Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md

```diff
@@ -136,3 +136,36 @@ Example:
 }
 <h3>@item.Value("heading")</h3>
 ```
+
+## Creating Nested Content programmatically
+
+For the sake of this example, let us assume we have a Nested Content property with alias `attendeeList`, where the element Document Type has an alias of `attendee`.
+It has the Properties: `user_name`, `user_email`, `join_time`, `leave_time`, `duration`, `phone`.
+
+To save data in Nested Content, we need to create a new C# List containing a `Dictionary` of type `<string, object>`. `Dictionary<string, string>` would also work.
+The first dictionary item property/parameter we should specify for each Nested Content element is `ncContentTypeAlias`, which is the alias of the Document Type that is the embed element in Nested Content.
+
+Afterwards, the entire list needs to be serialized to Json via JsonConvert.
+
+After injecting [ContentService](../../../../../Reference/Management/Services/ContentService/) and creating a new Content Node, we can do the following:
```

Comment:
Is this line necessary still, after you've added a variable for ContentService in the code sample?

Reply:
I suppose not - will delete it later

Reply:
Removed that line in the latest commit 👍